src/components/ImpactDialog.js

```diff
@@ -106,6 +106,30 @@ const ImpactDialog = ({
         </>
       )
       break
+    case 'newlyReferredImpactWalkMe':
+      dialogContent = (
+        <>
+          <MuiDialogTitle disableTypography className={classes.root}>
+            <Typography variant="h6" className={classes.title}>
+              You've already made a difference!
+            </Typography>
+          </MuiDialogTitle>
+          <MuiDialogContent className={classes.walkMe}>
+            <Typography className={classes.justify}>
+              When your friend referred you to Tab For Cats, you each earned
+              five treats to give to shelter cats. Now, when you open a new tab
```

Comment:
A few changes: "referred" is an internal term that's not very user-friendly; it's a bit long, and we already intro what treats do in the intro modal immediately previous to this; and I'd like to keep the "open a new tab" prompt to keep a clear next step. How about this:

### Open a new tab

Your friend started you off with 5 cat treats, which are crucial to getting shelter cats adopted.

Open a new tab now to earn your 6th treat! We'll track how many treats you've given on the top of the page:

Reply:
nice, yeah I like this better!

Reply:
ok tried messing around with the text being two sections with spacing but it never looked as good as one code block IMO.  I think it's b/c the text in each section is never just 1 line and they're similar length.  I like the language a lot better